server/queries/feedQuery.js

```diff
@@ -10,15 +10,15 @@ LEFT JOIN users u2 ON u2.id = c.author_id
 
 UNION ALL
 
```

Comment:
add a limit to this function so we don't get all the feeds at once. this will be a very slow process if we get. imagine we have two thousand users and they all made post and images, getting all this from the db is going to be slow. so add a limit to the query, may fetch the first 20 and display and then when a user scrolls on the frontend, you make another request to fetch the next 20 and offset. so you can research how to do this. it is called pagination

Reply:
Ok sir. I have tried to work something out concerning this pagination now